### PR TITLE
[esw, teleop] integrated robotic arm and science system controls

### PR DESCRIPTION
**Pull request by cgiger00**
Co-authored-by: Guthrie Tabios <gutstabios@gmail.com>
Co-authored-by: Cameron Tressler <camtressler@gmail.com>
Co-authored-by: Alan Tondryk <atondryk@umich.edu>
Co-authored-by: Jonathan Tom <jjtom@umich.edu>
Co-authored-by: Sashreek Shankar <sashreek123@hotmail.com>
Co-authored-by: Wilson Chen <77934980+wilsonchen02@users.noreply.github.com>
Co-authored-by: Joe Nanni <jnnanni@umich.edu>
Co-authored-by: ErikLytle <elytle@umich.edu>
Co-authored-by: Catherine Kim <catkims@umich.edu>
Co-authored-by: Sam Korman <skorman101@gmail.com>
Co-authored-by: Bryan Li <bryli@umich.edu>
Co-authored-by: Aidan Szuch <amszuch@umich.edu>

### CODE REVIEW

File: jetson/teleop/src/__main__.py
Comment (CameronTressler) [inline jetson/teleop/src/__main__.py:240]: Broken (wrong channel)

**Review by CameronTressler — CHANGES_REQUESTED** [review]

File: jetson/teleop/src/__main__.py
Comment (cgiger00) [reply jetson/teleop/src/__main__.py:240]: can you remove anything that's wrong?

File: ansible/roles/base_station_services/tasks/main.yml
Comment (tabiosg) [inline ansible/roles/base_station_services/tasks/main.yml:37]: So are we removing microscope or no?

Comment (cgiger00) [reply ansible/roles/base_station_services/tasks/main.yml:37]: yeah remove, did you push the microscope fix to this branch?

**Comment by cgiger00** [comment]
i'm pushing again to remove the raman folder...it'll rebuild but should still pass travis

File: ansible/roles/jetson_services/files/rover-jetson-raman.service
Comment (tabiosg) [inline ansible/roles/jetson_services/files/rover-jetson-raman.service:2]: so jetson/raman existed in the past? should delete maybe?

File: ansible/roles/base_station_services/tasks/main.yml
Comment (tabiosg) [reply ansible/roles/base_station_services/tasks/main.yml:37]: yes but i think this fix is why camera switching is soooo much slower

File: ansible/roles/jetson_services/files/rover-jetson-raman.service
Comment (cgiger00) [reply ansible/roles/jetson_services/files/rover-jetson-raman.service:2]: im deleting the raman folder, we will have a raman service when you guys get the port forwarding working

File: ansible/roles/base_station_services/tasks/main.yml
Comment (cgiger00) [reply ansible/roles/base_station_services/tasks/main.yml:37]: actually don't delete it quite yet...i actually still think we should have a microscope button since we can uniquely identify it by it's name

File: base_station/gui/src/components/Cameras.vue
Comment (tabiosg) [inline base_station/gui/src/components/Cameras.vue:5]: idk i plan on removing the microscope button... i would add to the todo to see about microscope stuff

File: ansible/roles/base_station_services/tasks/main.yml
Comment (tabiosg) [reply ansible/roles/base_station_services/tasks/main.yml:37]: sure

File: base_station/gui/src/components/Cameras.vue
Comment (cgiger00) [reply base_station/gui/src/components/Cameras.vue:5]: maybe don't, idk i think we should take advantage of being able to always identify it ideally soon, wouldn't make much sense imo to delete then add in

Comment (CameronTressler) [reply base_station/gui/src/components/Cameras.vue:5]: @catkims  is working on camera stuff now, so she could do it. But we should hold off on that for now?

Comment (CameronTressler) [reply base_station/gui/src/components/Cameras.vue:5]: Should we make this a draft to keep from running travis unnecessarily?

Comment (cgiger00) [reply base_station/gui/src/components/Cameras.vue:5]: smart, yes will convert to draft

**Review by CameronTressler — APPROVED** [review]

**Comment by CameronTressler** [comment]
Approving but waiting for Guthrie's PR (#1070)

**Comment by cgiger00** [comment]
latest commit should now let us pass travis without that PR if it takes more than a day, but i would prefer to have it in before merging

**Review by jjtom34 — APPROVED** [review]

**Review by tabiosg — APPROVED** [review]
looks good, i'll be working on some changes but i guess we can merge this in before then

**Comment by cgiger00** [comment]
ok merging

**Pull request merged**
